src/main/java/com/github/alexthe666/iceandfire/entity/ai/PixieAIPickupItem.java

```diff
@@ -41,9 +39,11 @@ public PixieAIPickupItem(EntityPixie creature, int chance, boolean checkSight, b
 
         this.targetEntitySelector = new Predicate<ItemEntity>() {
             @Override
-            public boolean apply(@Nullable ItemEntity item) {
+            public boolean test(ItemEntity item) {
 
-                return item != null && !item.getItem().isEmpty() && (item.getItem().getItem() == Items.CAKE && !creature.isTamed() || item.getItem().getItem() == Items.SUGAR && creature.isTamed() && creature.getHealth() < creature.getMaxHealth());
+                return !item.getItem().isEmpty() && (item.getItem().getItem() == Items.CAKE && !creature.isTamed()
```

Comment:
Same here

Reply:
Done here https://github.com/Alex-the-666/Ice_and_Fire/pull/4470/files#diff-139982523db614398c7410078f2b8f1d8f3149272d7349ca76b453afda0f7434R48

Reply:
Oh whoops didn't catch that sorry!